src/frontend/src/hooks/use-telescope-contributor.js

```diff
@@ -0,0 +1,30 @@
+import { useState, useEffect } from 'react';
+
+const useTelescopeContributor = () => {
+  const [contributor, setContributor] = useState(null);
+  const [loading, setLoading] = useState(false);
+  const [error, setError] = useState();
+
+  useEffect(() => {
+    const fetchContributors = async () => {
+      setLoading(true);
+      const response = await fetch(
+        'https://api.github.com/repos/Seneca-CDOT/telescope/contributors'
+      );
+      const data = await response.json();
+
+      if (data !== []) {
+        setContributor(data[Math.floor(Math.random() * data.length) + 1]);
```

Comment:
Move your logic for picking a random value from the array into a function that takes an array and returns an element from the array.